docs/guide/data_migration/data_migration.md

```diff
@@ -18,3 +18,9 @@ between installations by using the migration command.
     
     As of v3.3.3, the `ezmigrationbundle` package has been removed to mitigate this issue. 
     It is recommended that you use the default `ibexa/migrations` package to migrate your data. 
+
+
+## Examples of using migration
+
+You can find examples of using migrations in your project's
+`vendor/ibexa/migrations/tests/bundle/Command/MigrateCommand/migrate-command-fixtures` folder.
```

Comment:
```suggestion
You can find examples of using migrations in [Migration file content](exporting_and_importing_data.md#migration-file-content),
and in your project's `vendor/ibexa/migrations/tests/bundle/Command/MigrateCommand/migrate-command-fixtures` folder.
```
I suggest adding this as well, as started in https://github.com/ezsystems/developer-documentation/pull/1545/files#diff-da96e7f993350f34c0356c9d6b58ccd98bab222096cbc50dd49ce445d7407095R267